pkg/pc/fields/fields.go

```diff
@@ -58,6 +59,29 @@ type PodCluster struct {
 	Annotations Annotations
 }
 
+func (pc *PodCluster) Equals(other *PodCluster) bool {
+	if pc == nil && other == nil {
```

Comment:
although maybe some fields _shouldn't_ be included like if there's a status or something someday